models/ironbank/optimism/ironbank_optimism_sources.yml

```diff
@@ -7,31 +7,108 @@ sources:
       warn_after: { count: 12, period: hour }
```

Comment:
can you do me a favor and run the dbt source freshness command on all sources here? something like this (or close) should work:
```dbt source freshness --select +models/ironbank/optimism/```

if any return warnings or errors, they may need to either expand their timeframe or be removed all together

i will take note to try and include this in the gh action, so it's included as an automation test

Reply:
I got some error and I chose to remove them all.